tools/walletextension/main/cli.go

```diff
@@ -14,7 +14,7 @@ const (
 	walletExtensionPortUsage   = "The port on which to serve the wallet extension"
 
 	nodeHostName    = "nodeHost"
-	nodeHostDefault = "testnet.obscu.ro"
```

Comment:
Is the URL for testnet in any of the other readmes under docs? Might be worth adding the testnet URL into the Usage section of wallet-extension.md

Reply:
That's covered under https://docs.obscu.ro/testnet/essentials.html. One of the goals of the PR is to write a network-agnostic wallet extension docs, which isn't just about connecting to testnet.